clients/cobol-lsp-vscode-extension/esbuild.js

```diff
@@ -0,0 +1,99 @@
+const esbuild = require("esbuild");
```

Comment:
It looks like the `esbuild.js` name interferes with the esbuild package. I suggest renaming to e.g. `build.web.js`.

Reply:
Resolved in latest commit.